config/secrets.yml.example

```diff
@@ -30,6 +35,10 @@ test:
         hwIDAQAB
         -----END PUBLIC KEY-----
       encryption_private_key: 'RvGHVZ/kvzUAA5Z3t68+FNhuMCJxkzv+'
+  content:
+    archive_path: <%= ENV['OPENSTAX_CONTENT_ARCHIVE_PATH'] || 'apps/archive' %>
+    domain: <%= ENV['OPENSTAX_CONTENT_DOMAIN'] || 'openstax.org' %>
+  rex_host: <%= ENV['REX_HOST'] || 'https://dev.openstax.org' %>
```

Comment:
I didn't do this because `config.json` isn't setup for the local proxy, but after looking into it more, I see that it'd be easy to add it to the proxy and have it return `config.archive-url.json` (it only needs REACT_APP_ARCHIVE) what do you think?